shared/uavobjectdefinition/manualcontrolsettings.xml

```diff
@@ -105,13 +105,15 @@
 				<option>Always Armed</option>
 				<option>Roll Left</option>
 				<option>Roll Right</option>
-				<option>Pitch Forward</option>
-				<option>Pitch Aft</option>
 				<option>Yaw Left</option>
 				<option>Yaw Right</option>
+				<option>Corners</option>
 				<option>Switch</option>
+				<option>Switch+Throttle</option>
 			</options>
 		</field>
+		<field name="ArmTime" units="ms" type="enum" elements="1" options="250,500,1000,2000" defaultvalue="1000"/>
+		<field name="DisarmTime" units="ms" type="enum" elements="1" options="250,500,1000,2000" defaultvalue="1000"/>
```

Comment:
the timeout is still there, this is a different thing. it is how long you have to hold in the arm/disarm position for it to become active. The logic for actually disarming based on no transmitter is here https://github.com/TauLabs/TauLabs/pull/1482/files#diff-4f4804c22880c52150730a11f3c4a5baR636